src/sentry/search/utils.py

```diff
@@ -323,6 +323,9 @@ def tokenize_query(query):
     query_params = defaultdict(list)
     tokens = split_query_into_tokens(query)
     for token in tokens:
+        if token.upper() in ["OR", "AND"] or token.strip("()") == "":
```

Comment:
As far as I know, we don't use `tokenize_query` at all in issue search or discover search. Are we just changing this to mirror the code on the frontend?

Reply:
Yes. The two functions are supposedly linked (according to the comments) so I kept both doing the same logic.

Reply:
Ok, I think this is fine, but it does change the behaviour of other endpoints that use this in that OR and AND will be excluded from the tokenized query.

Reply:
I looked through all the uses, and all of them were using the tokenizer to find out if certain tags were being referenced, so excluding OR/AND doesn't have an effect on that.